package org.matteo.utils.concurrency.exception;

public interface ExceptionListener {

    void onException(Exception e);

}
